docs/modules/ROOT/pages/api/governance.adoc

```diff
@@ -1,9 +1,601 @@
 :github-icon: pass:[<svg class="icon"><use href="#github-icon"/></svg>]
+:CallScheduled: xref:ITimelock-CallScheduled[CallScheduled]
+:CallExecuted: xref:ITimelock-CallExecuted[CallExecuted]
+:CallSalt: xref:ITimelock-CallSalt[CallSalt]
+:CallCancelled: xref:ITimelock-CallCancelled[CallCancelled]
+:MinDelayChanged: xref:ITimelock-MinDelayChanged[MinDelayChanged]
+:RoleGranted: xref:api/access.adoc#IAccessControl-RoleGranted[IAccessControl::RoleGranted]
 
 = Governance
 
 Reference of interfaces and utilities related to Governance.
 
+== Timelock
+
+In a governance system, `TimelockControllerComponent` is in charge of introducing a delay between a proposal and its execution.
+
+[.contract]
+[[ITimelock]]
+=== `++ITimelock++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/release-v0.14.0/src/governance/timelock/interface.cairo[{github-icon},role=heading-link]
+
+[.hljs-theme-dark]
+```cairo
+use openzeppelin::governance::timelock::interface::ITimelock;
+```
+
+[.contract-index]
+.Functions
+--
+* xref:#ITimelock-is_operation[`++is_operation(id) -> bool++`]
```

Comment:
```suggestion
* xref:#ITimelock-is_operation[`++is_operation(id)++`]
```

We can reopen this discussion later, but for consistency we should remove the return values here.

Reply:
Whoops, agreed